jupyter_notebook_config.py

```diff
@@ -208,9 +208,18 @@
                 for chunk in r.iter_content(chunk_size=1024):
                     if chunk:  # filter out keep-alive new chunks
                         f.write(chunk)
-    if spark_mesos_uris:
-        spark_opts.append('--conf spark.mesos.uris={}'.format(
-            ','.join(spark_mesos_uris)))
+
+spark_conf_spark_mesos_uris = os.getenv('SPARK_CONF_SPARK_MESOS_URIS')
```

Comment:
Don't we have to expose `SPARK_CONF_SPARK_MESOS_URIS` via config.json/marathon.json.mustache as well?

Reply:
Yes, we do, this will be exposed in a follow on PR for DCOS-40456 while we land the pre-requisite support for it in this PR.